components/Directory/BreedCard/BreedCard.module.css

```diff
@@ -0,0 +1,15 @@
+.breedCard {
```

Comment:
This is a good start @nwhator, can you please convert this to Tailwind and use tailwind classes in the JSX? It's very commonly used and a good practice to learn. 

We can merge this for now but for the long-term we will be using Tailwind, so I will make another ticket to refactor this in that way :)